ignite/engine/events.py

```diff
@@ -348,81 +347,6 @@ def __or__(self, other: Union[Events, CallableEventWithFilter]) -> "EventsList":
         return self
 
 
-class State:
```

Comment:
@KickItLikeShika let's revert changes with the State as well. Thanks

Reply:
What about keeping it like this for now? My point is in this PR we don't do huge changes that requires a lot of reviews from your side, so I think moving State in this PR will prevent you from more headaches in next PR, and the next PR will be more focused on the important changes, what do you think?

Reply:
Let's do things by logical steps

Reply:
I see your point, I will revert state updates.